tide_media.module

```diff
@@ -321,9 +324,24 @@ function tide_media_views_pre_render(ViewExecutable $view) {
           if (strpos($file_mime, 'presentation')) {
             $value->_relationship_entities['field_media_file_target_id']->set('filemime', 'presentation');
           }
-          if (strpos($file_mime, 'text/plain')) {
+          if ($file_mime == 'text/plain') {
             $value->_relationship_entities['field_media_file_target_id']->set('filemime', 'plaintext');
           }
+          if ($file_mime == 'application/vnd.ms-excel.sheet.macroEnabled.12') {
```

Comment:
Are you sure this is correct MIME for Excel? It seems awfully specific

Reply:
hey @krakerag ,thanks for the CR
yes, it is not looking good! 
The mimetype for `.xlsb` file format is `application/vnd.ms-excel.sheet.macroEnabled.12`. Despite the awful name, we require it to obtain precise icons.

Reply:
OK - just wanted to double check. Carry on!